src/components/Onboarding/index.jsx

```diff
@@ -17,7 +35,9 @@ function Onboarding() {
           <br />
           디자인 화면의 변경사항을 쉽게 보여드려요!
         </span>
-        <Button size="lg">피그마 계정으로 로그인</Button>
+        <Button handleClick={onClickButtonHandler} size="lg">
```

Comment:
P4: 해당 버튼을 클릭하면 로그인을 할 수 있는 화면으로 넘어간다는 의미에서 Login이 들어가는 게 어떨까요? 

```suggestion
        <Button onClick={handleLogin} size="lg">
```

Reply:
버튼 컴포넌트를 재사용 하기 위해서 버튼 컴포넌트의 속성을 사용했습니다.